app/views/incomes/EstimatedPayLandingPageView.scala.html

```diff
@@ -30,47 +30,54 @@
 
 @header = @{
     ptaHeader(
-        displayBackLink = true,
+        displayBackLink = false,
         mainHeadingText = messages("tai.incomes.landing.Heading", incomeName),
+        showGdsVersion =true,
         preHeadingText = messages("tai.howToUpdate.preHeading", incomeName)
     )
 }
 
-@mainTemplate(
+@main(
     title = messages("tai.incomes.landing.title", incomeName),
     authedUser = Some(user),
-    pageHeader = Some(header),
-    articleClasses = None
+    pageTitle = Some(Messages("tai.service.navTitle")),
+    backLinkContent = Some(Messages("tai.back-link.upper")),
 ){
-    <div>
-        <p>@messages("tai.incomes.landing.intro")</p>
-        <p id="updateEstimatedPayTotalIncome" aria-hidden="true">@messages("tai.incomes.landing.employment.totalIncome", totalEstimatedIncome)</p>
-        <p id="updateEstimatedPayLink" aria-hidden="true"><a href="@routes.EstimatedIncomeTaxController.estimatedIncomeTax">@messages("tai.incomes.landing.employment.affect")</a></p>
-        <p>
-            @if(isPension) {
-                @messages("tai.incomes.landing.pension.ended", incomeName)
-                @link(
-                        url=appConfig.incomeFromEmploymentPensionLinkUrl,
-                        copy=messages("tai.incomes.landing.pension.ended.link"),
-                        id=Some("updatePension")
+    <div class="govuk-grid-row">
+        <div class = "govuk-grid-column-two-thirds">
+            @header
+        </div>
+        <div class="govuk-grid-column-two-thirds">
+            <p class="govuk-body">@messages("tai.incomes.landing.intro")</p>
+            <a id="updateEstimatedPayTotalIncome" aria-hidden="true" class="govuk-visually-hidden">@messages("tai.incomes.landing.employment.totalIncome", totalEstimatedIncome)</a>
```

Comment:
These 2 lines where we have visually hidden classes means they are for screen readers only - however they also have aria-hidden true which hides it from screen readers so there is a contradiction here I think  ???